views/index.slim

```diff
@@ -0,0 +1,5 @@
+div class="grid" id="transactions" data-url="/transactions" data-columns='["id","company","description","amount"]'
+  | loading ...
+
+div class="grid" id="trans_two" data-url="/transactions" data-columns='["id","company","email"]'
+  | loading ...
```

Comment:
@benrosenblum here you can see its the same endpoint but i'm specifying different columns. All that is required to get the library to work is the grid class and data-url and data-columns. The library defaults the sort to the first column listed. The column order is the order displayed.